### DISCUSSION TITLE
Setup storybook manually

### DISCUSSION THREAD

**Discussion by stevenKirill** [storybookjs/storybook#25475] [Help] [answered]
### Summary

Hello! Could somebody please help me, how can I setup storybook manually without sb init in react app with webpack config, which was manually set up ?

**Comment by valentinpalkovic** [comment, marked as answer]
You can take a look at https://github.com/storybookjs/sandboxes/tree/next/react-webpack/18-ts/after-storybook to install the right dependencies and setup the necessary `scripts`. Also, just copy over the `.storybook` folder. If you want to have some default stories in place, you can also copy over the following stories: https://github.com/storybookjs/sandboxes/tree/next/react-webpack/18-ts/after-storybook/src/stories.

Would you like to share why `npx sb@latest init` doesn't fit you?

**Reply by stevenKirill** [reply]
Thanks. I tried `npx sb@latest init`, but I got this error 
<img width="745" alt="Screenshot 2024-01-09 at 10 41 43" src="https://github.com/storybookjs/storybook/assets/47239173/7dd4ea20-8f10-4ed6-adcd-5f8c503fa9d8">

It seems that I have older version of babel installed in my package.json file. I have this fixed version "@babel/core": "7.16.0", and I don't want to upgrage it because I build my project using private registry, so I thought I can just install storybook manually without pain
